### PR TITLE
feat: styled application environment boxes

### PR DESCRIPTION
**Pull request by kwasniew**
## About the changes

Initial boxes for the successful application and list of environments:
Screenshot 2024-02-21 at 12 03 56

Next PR:
* connecting lines between the boxes
* warning box
* warning environment

### CODE REVIEW

**Review by sjaanus — APPROVED** [review]
LG, but I would have made styled components when using more than 1 sx attriute.

File: frontend/src/component/application/ApplicationOverview.tsx [hunk 1]
```diff
@@ -46,37 +85,92 @@ export const ApplicationOverview = () => {
         ],
     };
 
-    const applicationNode = `
-    ${applicationName}[<span>${applicationName}</span>]
-    `;
-
     // @ts-ignore
     window.navigateToInstances = (environment: string) => {
```
Comment (sjaanus) [inline frontend/src/component/application/ApplicationOverview.tsx:89]: This can also be removed now

Comment (sjaanus) [reply frontend/src/component/application/ApplicationOverview.tsx:89]: Btw, the bottom nodes should all navigate to relevant environment eventually.

**Comment by kwasniew** [comment]
@sjaanus I usually extract styled component after 3 sx params. But will refactor it anyway in the next PRs

File: frontend/src/component/application/ApplicationOverview.tsx (hunk 1 above)
Comment (kwasniew) [reply frontend/src/component/application/ApplicationOverview.tsx:89]: Yup Nico told me about this. We'll make those links in the next PR

**Pull request merged**